core/jazz_create-serverless-service/swagger/swagger.json

```diff
@@ -282,14 +282,13 @@
                     "type": "string",
                     "description":"Slack channel to receive all the service notifications"
                 },
-				"require_internal_access": {
+				        "require_internal_access": {
                     "type": "boolean",
                     "description":"Does service require access to internal SLF network and resources"
                 },
                 "deployment_targets": {
                     "type": "object",
                     "description": "Deployment targets list for the service",
-                    "required": false,
```

Comment:
Why did we remove this?

Reply:
that is not a valid swagger.

Reply:
You will still need to add this as required in some part of swagger. `deployment_targets` is a required field.

Reply:
updated